src/sbvr-api/sbvr-utils.coffee

```diff
@@ -826,16 +826,16 @@ exports.handleODataRequest = handleODataRequest = (req, res, next) ->
 # Reject the error to use the nice catch syntax
 constructError = (e) ->
```

Comment:
I wonder if we should change this so that we map internal errors to HTTPErrors and always respond with one of those